Hidden minimum values for sizes

Using version `0.1.1`.

The set Width and Height are positive, but probably not enough for the app to create a gif. The error is misleading.
```
➜ ./vhs shrug.tape
File: shrug.tape
Output .gif shrug.gif
Set FontSize 40
Set FontFamily Comic Mono
Set Width 300
Set Height 100
panic: {-32602 Width and height values must be positive, not greater than 10000000 }

goroutine 1 [running]:
github.com/go-rod/rod/lib/utils.glob..func2({0xd9ade0?, 0xc0002c6690?})
        /home/runner/go/pkg/mod/github.com/go-rod/rod@v0.112.0/lib/utils/utils.go:60 +0x25
github.com/go-rod/rod.genE.func1({0xc00011eda0?, 0x52fb0d?, 0x530b05?})
        /home/runner/go/pkg/mod/github.com/go-rod/rod@v0.112.0/must.go:36 +0x5a
github.com/go-rod/rod.(*Page).MustSetViewport(0xc000160370, 0x1?, 0x1?, 0x2?, 0x0?)
        /home/runner/go/pkg/mod/github.com/go-rod/rod@v0.112.0/must.go:307 +0xd8
main.(*VHS).Setup(0xc0004645c0)
        /home/runner/go/pkg/mod/github.com/charmbracelet/vhs@v0.1.1/vhs.go:91 +0x66
main.Evaluate({0xc0005a8000, 0x863}, {0x1454480, 0xc000132008}, {0x0, 0x0, 0x0?})
        /home/runner/go/pkg/mod/github.com/charmbracelet/vhs@v0.1.1/evaluator.go:48 +0x3fd
main.glob..func1(0x1a1b320?, {0xc0007356f0, 0x1, 0x1?})
        /home/runner/go/pkg/mod/github.com/charmbracelet/vhs@v0.1.1/main.go:60 +0x16d
github.com/spf13/cobra.(*Command).execute(0x1a1b320, {0xc000138010, 0x1, 0x1})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.0/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x1a1b320)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.0/command.go:1040 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.6.0/command.go:968
main.main()
        /home/runner/go/pkg/mod/github.com/charmbracelet/vhs@v0.1.1/main.go:135 +0x25
```

Working with wider sizes.
```
➜ ./vhs shrug.tape
File: shrug.tape
Output .gif shrug.gif
Set FontSize 40
Set FontFamily Comic Mono
Set Width 600
Set Height 200
Type ¯\_(ツ)_/¯
Space 10
Sleep 5s
Creating GIF...
```

--- a/evaluator.go
+++ b/evaluator.go
@@ -45,6 +45,11 @@ func Evaluate(tape string, out io.Writer, opts ...EvaluatorOption) error {
 		}
 	}
 
+	h, w, padding := v.Options.Video.Height, v.Options.Video.Width, v.Options.Video.Padding
+	if h < 2*padding || w < 2*padding {
+		return errors.New("height and width must be greater than " + fmt.Sprint(2*padding))
+	}
+
 	// Setup the terminal session so we can start executing commands.
 	v.Setup()
 

--- a/examples/demo.gif
+++ b/examples/demo.gif
@@ -1,3 +1,3 @@
 version https://git-lfs.github.com/spec/v1
-oid sha256:493d9f26e0e4798c6a3d6138f2213a3d32d66d3b38b70557fee465d572061d07
-size 32249
+oid sha256:f81190d928245e4b07fc4f3672857cce3bc19d7b9c85e36ae1cae2a6680e99e9
+size 30096

--- a/main.go
+++ b/main.go
@@ -29,9 +29,11 @@ var (
 	ttydMinVersion = version.Must(version.NewVersion("1.7.2"))
 
 	rootCmd = &cobra.Command{
-		Use:   "vhs <file>",
-		Short: "Run a given tape file and generates its outputs.",
-		Args:  cobra.MaximumNArgs(1),
+		Use:          "vhs <file>",
+		Short:        "Run a given tape file and generates its outputs.",
+		Args:         cobra.MaximumNArgs(1),
+		SilenceUsage: true,
+
 		RunE: func(cmd *cobra.Command, args []string) error {
 			err := ensureDependencies()
 			if err != nil {

